### ISSUE TITLE
Estimation of Resources and Output for Worklogs

### ISSUE THREAD

**Issue by khandelwal-aman** [khandelwal-aman/farmdata#11]
Add option to add estimated consumption and output of resources for work logs.

**Comment by khandelwal-aman** [comment]
Option to add estimated resources and output added for logs

**Comment by khandelwal-aman** [comment]
Log report needs a section for estimated resources and output for display

**Comment by Ssameer152** [comment]
Yes and now we will work on algorithm to calculate projection of actual vs estimate.
